packages/kbn-apm-synthtrace/src/lib/utils/create_pivot_transform.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+class PivotTransform {
```

Comment:
Given we are working with Streams here, the usage of name Transform seem misguiding.  wdyt ? its actually a factory no ?